### ISSUE TITLE
Guideline: Distinguishable

### ISSUE THREAD

**Issue by DonnaDuiker** [alkem-io/client-web#8370]
# Background
After going through the WCAG 2.1 checklist, creating issues for all A and AA points that still need work. 1 issue for each guideline.

This is guideline 1.4 Distinguishable

# Acceptance Criteria

- [ ] 1.4.3 Contrast (Minimum)	:Text has a contrast ratio of at least 4.5:1 (normal) or 3:1 (large text). (The muted color (soft grey) prbably doesn't have high eough contrast. Disabled buttons not. New colors is not AA)
- [ ] 1.4.4 Resize Text: Text can be resized to 200% without loss of functionality.	(More work needed, device setting, not zoom in in browser)
- [ ] 1.4.11 Non-text Contrast: UI components (buttons, icons) have a contrast ratio of at least 3:1. (Should be Checked, the disabled state?)
- [ ] 1.4.12 Text Spacing: Text spacing can be adjusted without breaking functionality.	(Not sure?)
- [ ] 1.4.13 Content on Hover or Focus: Additional content (tooltips) can be dismissed, hovered, or persisted. (Should be Checked, Is the tooltip even accessible, then check if it can be dismissed)

# Context

[Excel file](https://stichtingcherrytwist.sharepoint.com/:x:/s/Product/EawxIsWzBaRHt8K2E1tUwmsBzd93mWRRiImOP2M0muxujQ?e=eYV2Qr)

[W3C website](https://www.w3.org/TR/WCAG21/)
